tests/test_probability_premature_death.py

```diff
@@ -0,0 +1,36 @@
+
+import datetime
+import os
+from pathlib import Path
+
+import pandas as pd
+
+from tlo.analysis.probability_premature_death import get_probability_of_dying_before_70
+
+
+def test_get_probability_premature_death():
```

Comment:
given that we're running this test on a known data set, would it make sense to check that the actual numerical answer obtained is exactly equal to what we want?
(i realise the test you modelled this on, didn't do this, but arguably it should have!)